src/itest/java/uk/gov/companieshouse/insolvency/data/controller/InsolvencyControllerITest.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.companieshouse.insolvency.data.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.gson.Gson;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import uk.gov.companieshouse.insolvency.data.helpers.FixtureHelper;
+import uk.gov.companieshouse.insolvency.data.requests.InsolvencyRequest;
+import uk.gov.companieshouse.insolvency.data.service.InsolvencyService;
+
+import static org.mockito.Mockito.*;
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@ExtendWith(MockitoExtension.class)
+public class InsolvencyControllerITest {
+
+    private MockMvc mockMvc;
+
+    @Mock
+    private InsolvencyService insolvencyService;
+
+    @InjectMocks
+    InsolvencyController insolvencyController;
+
+    @Autowired
+    private FixtureHelper fixtureHelper;
```

Comment:
Try to use below to read from json. You dont have to create a bean.
@Value("classpath:data/resource-data.txt")
Resource resourceFile;